pypsa/optimization/constraints.py

```diff
@@ -363,50 +367,61 @@ def p_previous(idx):
             return reindex(p, c, idx).shift(snapshot=1).sel(snapshot=sns[1:])
 
     com_i = n.get_committable_i(c)
-    fix_i = n.get_non_extendable_i(c)
     fix_i = fix_i.difference(com_i).rename(fix_i.name)
```

Comment:
```suggestion
    fix_i = n.get_non_extendable_i(c)
    fix_i = fix_i.difference(com_i).rename(fix_i.name)
```
my fault when removing merge conflicts